src/brod_auth_backend.erl

```diff
@@ -0,0 +1,3 @@
+-module(brod_auth_backend).
```

Comment:
this module is also added here: https://github.com/ElMaxo/brod_gssapi/blob/master/src/brod_auth_backend.erl ?